Makefile.dep

```diff
@@ -564,6 +564,14 @@ ifneq (,$(filter random,$(USEMODULE)))
         USEMODULE += prng_tinymt32
     endif
 
+    ifneq (,$(filter prng_fortuna,$(USEMODULE)))
+        USEMODULE += fortuna
+        USEMODULE += hashes
+        USEMODULE += crypto
+        USEMODULE += xtimer
+        CFLAGS += -DCRYPTO_AES 
```

Comment:
I wonder if it would be more intuitive to set the CFLAG in `sys/random/Makefile`,
since here we only handle dependencies.
Beside, please remove the extra whitespace behind the flag.

Reply:
How about sys/Makefile.dep?